lstchain/scripts/lstchain_create_run_summary.py

```diff
@@ -0,0 +1,192 @@
+"""
+Create a run summary for a given date containing the number of subruns,
+the start time of the run, type pf the run: DATA, DRS4, CALI, and
+the reference timestamp and counter of the run.
+"""
+
+import argparse
+import logging
+from pathlib import Path
+
+import numpy as np
+import protozfits
+from astropy.table import Table
+from astropy.time import Time
+from ctapipe.containers import EventType
+from ctapipe.io import EventSource
+from lstchain.paths import parse_r0_filename
+
+# FIXME: take it from somewhere else
+tel_id = 1
+
+
+log = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(description="Create run summary file")
+
+parser.add_argument(
+    "-d",
+    "--date",
+    type=str,
+    help="Date for the creation of the run summary in format YYYYMMDD",
+    required=True,
+)
+# TODO: default today()
+
+parser.add_argument(
+    "--r0-path",
+    type=Path,
+    dest="R0_PATH",
+    help="Path to the R0 files. Default is /fefs/aswg/data/real/R0",
+    default=Path("/fefs/aswg/data/real/R0"),
+)
+
+parser.add_argument(
+    "-o",
+    "--output-dir",
+    type=Path,
+    dest="output_dir",
+    help="Directory in which Run Summary file is written",
+    default="./",
+)
+
+
+args = parser.parse_args()
+
+
+def get_list_of_files(date):
+    """Get the list of R0 files from a given date."""
+    path_r0 = args.R0_PATH / date
+    # FIXME: use regular expressions from lstchain.paths.R0_RE
+    list_of_files = path_r0.glob("LST*.fits.fz")
+    return list_of_files
+
+
+def get_list_of_runs(list_of_files):
+    """Get the sorted list of run objects from R0 filenames."""
+    return sorted(parse_r0_filename(file) for file in list_of_files)
+
+
+def get_runs_and_subruns(list_of_run_objects):
+    """
+    Get the list of run numbers and the number of sequenctial files (subruns)
+    of each run for a given date assuming 4 ZFW streams.
+    """
+    NUMBER_OF_STREAMS = 4
```

Comment:
What will happen with <4 - streams data?

Reply:
> What will happen with <4 - streams data?

I mean for 3-stream data you have to rename files to make them look like 4-stream ones. But 2-stream data would not need that trick. Would this work for 2-streams data?

Reply:
Not really. I can count the number of occurrences for a given run and a given stream (the first one) for example.

Reply:
As far as I know we do not have 2-streams data yet, so you may just add a comment now on that, and defer this until the moment we need it, or  in any case until after the coming release.

Reply:
@moralejo, 
if we want to test the 2 streams case, we have some test (pulser) data: run 3560